### PR TITLE
Updating Dolma's shopping list for the party 

### PR DESCRIPTION
**Pull request by DolmaMi**
This is the fix to issue #10.
@pbuttigieg would you mind reviewing the fix?

### CODE REVIEW

File: lists/DolmaShoppingList.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-Potatoes
-Leak
-Cheese
-Flowers
+Potatoes 
+Leak
```
Comment (pbuttigieg) [inline lists/DolmaShoppingList.md:2]: I assume you don't need a plumber? Leek right?

File: lists/DolmaShoppingList.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
-Potatoes
-Leak
-Cheese
-Flowers
+Potatoes 
+Leak 
+Cheese 
+Flowers
```
Comment (pbuttigieg) [inline lists/DolmaShoppingList.md:4]: Please be more specific 🥀

File: lists/DolmaShoppingList.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
-Cheese
-Flowers
+Potatoes 
+Leak 
+Cheese 
+Flowers 
+Chocolate
+Lamb
+Rosemary
+Wine
```
Comment (pbuttigieg) [inline lists/DolmaShoppingList.md:8]: Again, which box? Which colour?

**Review by pbuttigieg — CHANGES_REQUESTED** [review]
Hi @DolmaMi just a typo and need for more celebratory precision in your list

**Comment by pbuttigieg** [comment]
@DolmaMi the review changes above are minor, please resolve them yourself when solved.

File: lists/DolmaShoppingList.md (hunk 2 above)
Comment (DolmaMi) [reply lists/DolmaShoppingList.md]: resolved in https://github.com/pbuttigieg/shoppingList/pull/17/commits/25c7b3cbce8c47765a802b9176680890db228a59

File: lists/DolmaShoppingList.md (hunk 3 above)
Comment (DolmaMi) [reply lists/DolmaShoppingList.md]: resolved in https://github.com/pbuttigieg/shoppingList/pull/17/commits/25c7b3cbce8c47765a802b9176680890db228a59

**Comment by pbuttigieg** [comment]
@DolmaMi looks good thanks! Go ahead and merge.

**Review by pbuttigieg — APPROVED** [review]

**Pull request merged**